### PR TITLE
fix(youtube): premiere badge is too low contrast

### PR DESCRIPTION
**Pull request by BPplays**
## 🔧 What does this fix? 🔧

the premiere badge to indicate a video is a premiere was colored with light text color and light accent color it's now a dark text color

## 🗒 Checklist 🗒

- [x] I have read and followed Catppuccin's [contributing guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/CONTRIBUTING.md).
- [x] I have updated the version appropriately in the `==UserStyle==` header of the `catppuccin.user.css` file.

### CODE REVIEW

File: styles/youtube/catppuccin.user.css [hunk 1]
```diff
@@ -382,6 +382,9 @@
       --yt-endpoint-visited-color: @accent-color !important;
       color: @accent-color !important;
     }
+    .badge-style-type-live-now-alternate.ytd-badge-supported-renderer[aria-label="PREMIERE"] {
+      color: @surface0 !important;
+    }
```
Comment (isabelroses) [inline styles/youtube/catppuccin.user.css:387]: The premier badge was added here, https://github.com/catppuccin/userstyles/pull/897/files#diff-c08ba2f30a7396e0d274fcbc429a1616b4dd4373f815d47f2b082ef52dbf4928R698. Do you mind changing this value instead,

Comment (BPplays) [reply styles/youtube/catppuccin.user.css]: changed it

**Comment by uncenter** [comment]
Unable to reproduce? Where are you seeing this?

**Comment by BPplays** [comment]
@uncenter i saw it on a premiere video in my recommended
before:
before
after:
after

**Comment by uncenter** [comment]
Pretty sure I fixed this in #897 but I can't find a premiere video on my home page to test with. Are you sure you were up to date before making this PR?

**Comment by BPplays** [comment]
@uncenter dang i think i might have been slightly out of date the version i was on was before that commit

**Comment by uncenter** [comment]
No worries ahah. Stylus auto updater sucks (tbh I'm not even sure if it is real at this point).

**Pull request closed without merging**